lib/customer_io.dart

```diff
@@ -1,5 +1,7 @@
 import 'dart:async';
 
+import 'package:customer_io/messaging_in_app/platform_interface.dart';
+
```

Comment:
I think we should keep imports consistent. e.g.
```
import 'messaging_in_app/platform_interface.dart';
import 'messaging_push/platform_interface.dart';
```
OR
```
import 'package:customer_io/messaging_in_app/platform_interface.dart';
import 'package:customer_io/messaging_push/platform_interface.dart';
```

I would prefer the first one.